openpype/hosts/nuke/plugins/publish/extract_review_intermediates.py

```diff
@@ -8,15 +8,15 @@
 from openpype.hosts.nuke.api.lib import maintained_selection
 
 
-class ExtractReviewDataMov(publish.Extractor):
-    """Extracts movie and thumbnail with baked in luts
+class ExtractReviewIntermediates(publish.Extractor):
+    """Extracts Sequences and thumbnail with baked in luts
```

Comment:
This plugin is not only for generating Intermediate Sequences. This is misleading and I would rather recommend 

```suggestion
    """Extracting intermediate files for transcoding. 
```

Reply:
If it only does videos or sequences I think mentioning that is clearer than "files" by the way - I'd not expect it to process word documents or audio files? :) Does it do single images?

Reply:
I edited the docsting to """Extracting intermediate videos or sequences with thumbnail for transcoding"""